README.md

```diff
@@ -182,6 +182,27 @@ If you don't have a scanner on your phone, please install:
 
 - [ST BLE Profile](https://play.google.com/store/apps/details?id=com.stm.bluetoothlevalidation) for Android.
 
+
+## Using bare metal profile
+
+Example with NUCLEO_WB55RG:
+
+mbed_app.json:
+
+```
+{
+    "requires": ["bare-metal", "events", "cordio-stm32wb"],
+    "target_overrides": {
+        "*": {
+            "target.c_lib": "small",
```

Comment:
These are not specific to bare metal but help reduce further the size of the application, can you split the description in two: 

One for the require clause that enables the bare-metal profile and another for std IO optimization.